### ISSUE TITLE
Dropdown: dissociate focused and hovered states

### ISSUE THREAD

**Issue by ffoodd** [Orange-OpenSource/Orange-Boosted-Bootstrap#643]
Hover state should stick to its black background, but focus state could rely on some gray (`#eee` mentioned on Mattermost).

Should be asked for review to GDL, at some point.

**Comment by ffoodd** [comment]
After a few back-and-forth with EASE, it seems our current v4 moving focus to the first item doesn't match the [ARIA Disclosure design pattern](https://www.w3.org/TR/wai-aria-practices/#disclosure):
- WAI disclosure example: https://www.w3.org/TR/wai-aria-practices/examples/disclosure/disclosure-navigation.html
- Atalan's AcceDe Web guideline: https://www.accede-web.com/en/guidelines/rich-interface-components/drop-down-menu/
- Orange a11y guidelines: https://a11y-guidelines.orange.com/en/web/components-examples/dropdown-menu/

So I'd recommend to stop moving focus around.
